lib/practice_exercises.rb

```diff
@@ -1,13 +1,37 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
```

Comment:
Since `delete_at` ends up shifting all subsequent elements to the left by one index, and you're putting it in a loop which runs n times, this is actually O(n<sup>2</sup>)